### ISSUE TITLE
(Enhancement?) Ability to cache response as a different content_type from the Browseable API

### ISSUE THREAD

**Issue by njamaleddine** [chibisov/drf-extensions#76]
My cached responses were storing `text/html` when one might want the ability to store `application/json`. 

Perhaps we can have a `content_type` passed in through the kwargs of `@cache_response()`. Apologies if I missed something in the docs or code by accident. 

I am aware of the global setting below and the ability to pass `content_type`s into a Response object or rendered response in Django REST Framework.

`settings.py`:
```
REST_FRAMEWORK['DEFAULT_RENDERER_CLASSES'] = (
    'rest_framework.renderers.JSONRenderer',
)
```

**Comment by chibisov** [comment]
I couldn't understand your case. Maybe [FormatKeyBit](http://chibisov.github.io/drf-extensions/docs/#formatkeybit) will help you? Could you describe your problem in more details?

**Comment by njamaleddine** [comment]
My apologies I realized that I had an error in the key function. I was creating the same key for the list method regardless of the client. So when I was viewing the API in the browser it was storing the response data in `text/html`. And when I viewed it with something like curl it was showing the cached response, grabbing the same key and showing `text/html`. I'll have to modify my key so it stores the right `content_type`. 

Thanks. I will take a look at FormatKeyBit.
